pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandler.java

```diff
@@ -93,12 +94,10 @@ public class ForwardIndexHandler extends BaseIndexHandler {
   private final Schema _schema;
 
   protected enum Operation {
```

Comment:
(optional) Ideally we can remove the `Operation` enum, and update the index in steps. Can be handled separately

Reply:
Sure, let me pick this up as a future fix (along with the changes to remove some of the tests that aren't really needed)